### ISSUE TITLE
dialog restart bug when selecting a specific option - Unity dialog with mutant

### ISSUE THREAD

**Issue by tastez** [rotators/Fo1in2#226]
in the current version there is a dialog option which will flicker the UI and restart dialog with the super mutant Lou

**Comment by Lexx2k** [comment]
Need way more information than that. You could at least write which dialog option that is and provide a savegame. Otherwise nobody will ever bother checking it out.

**Comment by tastez** [comment]
hey i figured out the issue after choosing another dialog option and getting smahed in the face and restarting the dialog and again going to those 2 options - 
when you ask Lou to tell you more about Unity and then at the end of the dialog you can say that it sounds dillusional or how are the mutants hoping to bring peace or something, the game exits the dialog, Lou smacks you, and then dialog restarts.

What happens if you do this option the first time around, the game doesnt exit dialog and doesnt show you getting smacked - you just get the dialog restart again.

https://github.com/rotators/Fo1in2/assets/78642112/b6409439-8bd0-41ca-9b15-f007b8e03472

**Comment by Lexx2k** [comment]
How high is your characters intelligence?

**Comment by tastez** [comment]
![Fallout2_7wkQ72yar9](https://github.com/rotators/Fo1in2/assets/78642112/b14803a9-9519-44a9-968e-175f8f5ee719)
is it because im bald?

**Comment by Lexx2k** [comment]
Nah. It's your 5 INT. Found the issue earlier, just wanted a confirmation. Not sure yet how to deal with it, need to think about it some more.
